scripts/reset-dockstoretestuser4-resources.sh

```diff
@@ -0,0 +1,77 @@
+#!/bin/bash
+
+STACK=$1
+
+if [ "$STACK" == "qa" ]
+then
+  URL="https://qa.dockstore.org"
+  TOKEN=${CYPRESS_QA_TOKEN}
+elif [ "$STACK" == "staging" ]
+then
+  URL="https://staging.dockstore.org"
+  TOKEN=${CYPRESS_STAGING_TOKEN}
+else
+  URL="https://dockstore.org"
```

Comment:
this is a little undesirable to have prod as a default, could imagine someone adding a new environment in .circleci/config.yml and forgetting to modify this or mis-spelling something and hitting prod 
It's a test user, so its not a big deal but would prefer to have this just fail on unknown to make it obvious